v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,89 @@
+industry-leading network policy enforcement with the flannel overlay and
+non-overlay network connectivity options.
+
+Note that the Canal currently uses the Calico and flannel projects as is with
+no code modifications to either. Canal is simply a deployment pattern
+for installing and configuring the projects to work together seamlessly as a
+single network solution from the point of view of the user and orchestration
+system.
+
+
+## Installation
+
+When deploying your kubernetes cluster, please make sure it meets the
+[requirements](#requirements--limitations) at the bottom of this page.
+An easy way to create a cluster which meets these requirements is by following
+[the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+
+> **Note:** If you are upgrading from the Kubernetes 
+[1.6](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#for-kubernetes-16)
+or [1.5](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#kubernetes-15)
+manifests from the (deprecated) Canal repo to the manifests here it is
+neccessary to [migrate your Calico configuration
+data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md)
+before upgrading. Otherwise, your cluster may lose connectivity after the 
+upgrade.                       
+
+### RBAC 
+  
+Before you install Canal, if your Kubernetes cluster has RBAC enabled, you'll
+need to create the following RBAC roles to allow API access to Calico.
+
+Apply the following manifest to create these necessary RBAC roles and bindings.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/rbac.yaml
+```
+> You can also [view the YAML in your browser.](https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/rbac.yaml)
+ 
+### Canal with Kubernetes Datastore
+
+The recommended Canal installation uses the Kubernetes API as the datastore,
+the manifest below installs Calico and Flannel configured to use the
+Kubernetes API.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/canal.yaml
+```
+> You can also [view the YAML in your browser.](https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/canal.yaml)
+
+
+### (or) Canal with Etcd Datastore
+
+We strongly recommend using the Kubernetes API manifests above, but if you
+have a need to use etcd we have provided an example etcd with TLS manifest
+[`canal_etcd_tls.yaml`](1.7/canal-ets.yaml). 
+
+When using an etcd datastore, the provided manifest allows you to specify
+the etcd endpoints for your etcd cluster, which must be configured
+independently.
+
+By default, the manifest expects an etcd proxy to be running on each
+Kubernetes node at `http://127.0.0.1:2379`.
+
+
+### Requirements / Limitations
+
+- This install assumes no other pod network configurations have been installed
+  in /etc/cni/net.d (or equivilent directory).
```

Comment:
You have a misspelled word here. "equivilent" is not a word...should be "equivalent" Thanks!